libafl/src/executors/timeout.rs

```diff
@@ -296,11 +358,10 @@ where
         mgr: &mut EM,
         input: &I,
     ) -> Result<ExitKind, Error> {
-        #[cfg(unix)]
         unsafe {
             setitimer(ITIMER_REAL, &mut self.itimerval, null_mut());
             let ret = self.executor.run_target(fuzzer, state, mgr, input);
-            unix_remove_timeout();
+            self.reset_on_crash();
```

Comment:
Wait, if it's called here, then `reset_on_crash` doesn't seem consistent, sorry!
Maybe `post_run`?